extra/release_tool.py

```diff
@@ -2863,15 +2868,15 @@ def do_set_version_to(args):
         set_component_version_to(integration_dir(), component, args.version)
         # Update extra files used in integration tests
         set_component_version_to(
-            os.path.join(integration_dir(), "extra", "mtls"), component, args.version,
-        )
-        set_component_version_to(
-            os.path.join(integration_dir(), "extra", "failover-testing"),
+            os.path.join(integration_dir(), "backend-tests", "docker"),
             component,
             args.version,
         )
         set_component_version_to(
-            os.path.join(integration_dir(), "extra", "mender-gateway"),
```

Comment:
Why remove this? Isn't it present anymore?

Reply:
it's a weird diff, I simply added backend-tests and sorted them to match in the two portions of code.